src/plugins/intel_cpu/src/nodes/common/cpu_convert.cpp

```diff
@@ -516,6 +537,13 @@ bool isConversionTruncatesRange(const Precision & from, const Precision & to) {
     MKLDNN_CVT(FP32, FP32), MKLDNN_CVT(FP16, FP16), MKLDNN_CVT(BF16, BF16), MKLDNN_CVT(FP64, FP64), \
     MKLDNN_CVT(BOOL, BOOL)
 
+#define MKLDNN_COPY(DT) OV_CASE(Precision::DT, PrecisionInfo<Precision::DT>::value_type)
+#define MKLDNN_COPY_LIST \
+    MKLDNN_COPY(U8),    MKLDNN_COPY(I8),     MKLDNN_COPY(U16),   MKLDNN_COPY(I16),   \
```

Comment:
To reduce binary size impact I would propose not to create instance for each precision but split it based on precision size. There is no difference in behavior when you copy U8 or I8 data since cast part is absent.

Reply:
Yes, agree with you and I change the parallel-copy to type-agnostic  which doesn't use template